plugins/analysis/client/plugin.go

```diff
@@ -73,9 +93,36 @@ func run(_ *node.Plugin) {
 			case <-ticker.C:
 				sendHeartbeat(conn, createHeartbeat())
 				sendMetricHeartbeat(conn, createMetricHeartbeat())
+				manaHb, err := createManaHeartbeat()
```

Comment:
I think we need more fine grained control over what a client is sending. In this form, all clients will flood the server with mana hb packets every 5s. Since we only need one reliable source in the analyzer,  sending mana hb packets should be disabled by default, and has to be explicitly enabled in config. On top of course we can discard mana hb packets from other nodes. My original thinking was that we don't even need the mana hb packets, why don't we just execute api calls to a node we configure from the anaylzer server? That seem to be less hassle, and analysis client can stay as it is. Why add more code and logic if we already have the api calls for this? (except for the events, true) 
What if we connect the analyzer dashboard frontend to the WS msg stream of the local dashboard of a node we choose? We would have all the mana data without using either the API or the heartbeats.

Reply:
>What if we connect the analyzer dashboard frontend to the WS msg stream of the local dashboard of a node we choose?

That's a great idea.